testing/src/main/java/org/jdbi/v3/testing/junit5/JdbiSqliteExtension.java

```diff
@@ -61,7 +65,7 @@ public JdbiSqliteExtension() {}
 
     @Override
     public String getUrl() {
-        return URL;
+        return String.format(URL_FMT, id);
```

Comment:
I think a better fix for this would be e.g. what we do with the `GenericJdbiExtension` and add a `withUrl()` method or a c'tor that takes an URL. So that you can do 

```java
@RegisterExtension
    public JdbiExtension sqlLite = new JdbiSqliteExtension("... specific URL ...");
```

or
```java
@RegisterExtension
    public JdbiExtension sqlLite = new JdbiSqliteExtension()
        .withUrl("... specific URL ...");
```

See e.g. JdbiGenericExtensionTest.